cdhweb/resources/models.py

```diff
@@ -82,8 +78,9 @@ def __str__(self):
 class LandingPage(Page, RichText):
     tagline = models.CharField(max_length=255)
     image = FileField(verbose_name="Image",
-        upload_to=upload_to("resources.landing_pages.image", "resources"),
-        format="Image", max_length=255, null=True, blank=True)
+                      upload_to=upload_to(
+                          "resources.landing_pages.image", "resources"),
+                      format="Image", max_length=255, null=True, blank=True)
 
 
 class PublishedQuerySetMixin(models.QuerySet):
```

Comment:
This will go away, right? Should we add a deprecated warning or will it naturally be removed once we exodize all the relevant pieces?

Reply:
We could add a deprecation warning, but I think it will naturally stop being used once we exodize all the relevant pieces, at which point we can just delete. The wagtail equivalent is just `.live` (on a page model) or `.live()` (on a queryset of pages).